astropy/units/tests/test_quantity_non_ufuncs.py

```diff
@@ -674,13 +674,13 @@ def test_all(self):
         with pytest.raises(TypeError):
             np.all(self.q)
 
-    # NUMPY_LT_1_25
```

Comment:
Please add `# NUMPY_LT_1_25` back. Pretty sure Marten added them there so we can grep for this string when it is time to drop NumPy 1.25.

Reply:
Update: See https://github.com/astropy/astropy/pull/16062#discussion_r1495067574

Reply:
`# NUMPY_LT_1_25` should then remain ahead of the `filterwarnings`, not the `skipif` (is there no way to make the `filterwarnings` conditional as well?) "NumPy 1.25" is also still easy to grep, if we just remember it...

Reply:
Since Marten is okay with the removal, I will merge when devdeps is green. I restarted it to make the remote data failure go away, just to be sure. Thanks, all!